tensorflow/python/ops/math_ops.py

```diff
@@ -334,10 +334,28 @@ def divide(x, y, name=None):
 
 @tf_export("math.multiply", "multiply")
 @dispatch.add_dispatch_support
-def multiply(x, y, name=None):
+def multiply(x, y, name=None):  # pylint: disable=missing-docstring
```

Comment:
The pylint comment is not required here.
Also, the comment on line below. Please remove changes outside of the docstring `"""..."""`.

Reply:
Rest looks perfect :) ! :+1: